internal/store/group.go

```diff
@@ -10,15 +10,42 @@ import (
 
 var groupSelect sq.SelectBuilder
 
+type rawGroup struct {
+	ID            string
+	Name          string
+	Description   string
+	Version       string
+	CreateAt      int64
+	DeleteAt      int64
+	MattermostEnv []byte
+}
```

Comment:
With a slightly different tactic, you can just embed the group struct directly here. The benefit of this is that you don't need to remember to update this when adding new group values. Also, tests pass when doing it this way without modification.

Example: https://github.com/mattermost/mattermost-cloud/pull/161/files#diff-f9b9c67a53c130978d472f7bb6412687R24-R27

Reply:
Hmm, I thought there was some reason that I didn't just embed. Maybe not. I'll give this a shot, thanks.

Reply:
Ah I see, it was a silly naming problem that was preventing me from doing this in the first place.